src/pages/users/users-details/containers/user-details-card/user-details-card.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import {
+  Card,
+  CardContent,
+  Grid,
+  Avatar,
+  Typography,
+  Button
+} from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { useStyles } from './user-details-card.styles';
+import { config } from '../../../../../configs';
+
+const userTitles = config.detailTitles.users;
+
+const UserDetailsCard = ({
+  avatar,
+  name,
+  status,
+  primaryData,
+  secondaryData,
+  buttonStatus,
+  buttonHandler
+}) => {
+  const styles = useStyles();
+
+  const buildInfoItem = (label, id, data, key) => (
```

Comment:
Please, move it to the separate component

Reply:
fixed